source/apiVolontaria/volunteer/tests/tests_view_ParticipationsId.py

```diff
@@ -118,7 +148,7 @@ def test_retrieve_participation_id_not_exist(self):
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
         self.assertEqual(json.loads(response.content), content)
 
-    def test_retrieve_participation(self):
+    def test_retrieve_participation_has_owner(self):
```

Comment:
test_retrieve_participation_**as**_owner ? Keep in mind that changes that are not related to your PR must be kept to a minimum and be necessary.

Reply:
All the PR is playing with permissions but I understand the borders.  Do you advise removing this test ?

Reply:
@yanicolivier Why would we remove this test?

Reply:
@francisjeanneau Oups, sorry, I misunderstood your sentence.  I have corrected the type. :)